packages/seacas/libraries/ioss/src/Ioss_Utils.C

```diff
@@ -2257,7 +2257,7 @@ namespace {
       for (const auto &set : sets) {
         const std::string &name = set->name();
         if (options.debug && rank == 0) {
-          fmt::print(Ioss::DEBUG(), "{}, ", name);
+          fmt::print(stderr, "{}, ", name);
```

Comment:
The 'stderr' should be 'Ioss::DEBUG()'

Similar for some locations down below.

Reply:
This is another cherry pick merge error.  I fixed these plus two other "stderr" uses that I found in `transfer_assemblies()` and `transfer_blobs()` (commit 135650d).